dts/bindings/spi/nordic,nrf-spi-common.yaml

```diff
@@ -15,14 +15,14 @@ properties:
     sck-pin:
       type: int
       required: true
-      description: SCK pin
+      description: SCK pin number, passed to nrf_gpio_pin_port_decode()
```

Comment:
Its a little odd to document this w/a reference to an API in a HAL.  Is the `pin number` not something documented in the user manuals?

Reply:
Since this is shared across multiple bindings, I'd have to refer to multiple product specifications here, and that would go stale.

Reply:
Is this about how pin numbers are specified in general for nRF devices? because if that is the case then I think we should document this everywhere. The same mechanism is presumably used in every peripheral and in the gpio bindings?

Reply:
I'm not convinced that such reference to `nrf_gpio_pin_port_decode()` is the best way of explaining it. Maybe providing a formula for calculation of this number from the Px.y form would be better? I found something like this in [nrf52840_pca10056 documentation](https://github.com/zephyrproject-rtos/zephyr/blob/master/boards/arm/nrf52840_pca10056/doc/index.rst#selecting-the-pins).

Reply:
@anangl what do you think about the latest update?